segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/ZKSegmentContainerMonitor.java

```diff
@@ -106,9 +107,16 @@ public void initialize() {
     public void initialize(Duration monitorInterval) {
         Exceptions.checkNotClosed(closed.get(), this);
 
+        // Start loading the segment container to node assigment map from zookeeper.
         this.hostContainerMapNode.start();
+
+        // There are two triggers for the segment container monitor.
+        // 1. On any segment container ownership changes notified via zookeeper. We will ensure the local containers
+        //      are stopped/started according to the new ownership mapping.
+        // 2. At scheduled intervals to perform retries on local segment container start failures.
         this.assigmentTask.set(this.executor.scheduleWithFixedDelay(
```

Comment:
I suppose we still need this line because of failures to start a container, but I'm wondering if this is redundant given the line you're adding below (line 119). I'm saying that this is redundant because it will be executing this in regular intervals.